packages/leancode_lint/README.md

```diff
@@ -0,0 +1,44 @@
+# leancode_lint
+
+[![leancode_lint pub.dev badge][pub-badge]][pub-badge-link]
+
+Lint rules used internally in LeanCode projects.
+
+## Usage
+
+Add `leancode_lint` as normal dependency to access `unawaited` function.
```

Comment:
```suggestion
Add `leancode_lint` as a normal dependency to access the `unawaited` function.
```

Reply:
Also, it's worth mentioning that since Dart 2.14 `unawaited` is available in `dart:async`: https://api.dart.dev/stable/2.14.2/dart-async/unawaited.html